### PR TITLE
Add (e)RPM black box field

### PR DESCRIPTION
**Pull request by tbolin**
Adds formatting of the RPM field from https://github.com/betaflight/betaflight/pull/12823
The field should show up as just RPM and be decoded as RPM and Hz.
The RPM plots are scaled so that they have the correct height relative to each other, so the same height corresponds to the same RPM for all plots.
Also adds an option in Configure graphs to add a graph with all RPM plots.

rpm_bb_field
bb_erpm_example_graph

<hr>
Depends on https://github.com/betaflight/betaflight/pull/12823

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> FAIL
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by nerdCopter** [comment]
@tbolin , Can you update description with details and screencap for prosperity sake? TY

**Comment by tbolin** [comment]
I've added a bit of detail and some screenshots

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
